prime-router/src/main/kotlin/fhirengine/utils/HL7Reader.kt

```diff
@@ -126,7 +127,9 @@ class HL7Reader(private val actionLogger: ActionLogger) : Logging {
                         v27_ORU_R01::class.java,
                         v251_ORU_R01::class.java
                     )
-
+                    "OML" -> listOf(
```

Comment:
ORM should also be added. I just tried processing an ORM message and got the warning.

Reply:
@victor-chaparro Any objections to me making that part of a separate PR? I was going to add it, but attempting to do so is inexplicably causes unit tests to fail:

```
Failed to map supported failure 'org.opentest4j.AssertionFailedError: expected to be true' with mapper 'org.gradle.api.internal.tasks.testing.failure.mappers.OpenTestAssertionFailedMapper@17bc1586': Cannot invoke "Object.getClass()" because "obj" is null

> Task :test
gov.cdc.prime.router.fhirengine.utils.FHIRBundleHelpersTests Test enhance bundle metadata 2-7() FAILED

  org.opentest4j.AssertionFailedError: expected to be true
      at app//gov.cdc.prime.router.fhirengine.utils.FHIRBundleHelpersTests.Test enhance bundle metadata 2-7(FHIRBundleHelpersTests.kt:672)


FAILURE: Executed 1097 tests in 1m 36s (1 failed, 4 skipped)
```